spec/features/admin/payments_stripe_spec.rb

```diff
@@ -82,7 +82,7 @@
 
           expect(page).to have_link "StripeSCA"
           expect(page).to have_content "PENDING"
```

Comment:
This test looks a bit weird now. Should it say "pending" in the UI here?

Also the test description is "fails to add a payment due to card error", maybe that needs updating?

Reply:
Updated 👍 ccc873fd25eaa42be8875556c4f1195318843296